packages/ui/react-ui-stack/src/components/Section.tsx

```diff
@@ -38,20 +40,25 @@ export type SectionProps = PropsWithChildren<{
 }>;
 
 export const Section = forwardRef<HTMLLIElement, SectionProps>(
-  ({ id, title, active, draggableProps, draggableStyle, onRemove, onNavigate, children }, forwardedRef) => {
+  (
+    { id, title, index, count, active, draggableProps, draggableStyle, onRemove, onNavigate, children },
+    forwardedRef,
+  ) => {
     const { t } = useTranslation(translationKey);
     const [optionsMenuOpen, setOptionsMenuOpen] = useState(false);
 
     return (
       <DensityProvider density='fine'>
-        <ListItem.Root ref={forwardedRef} id={id} classNames='pbe-2 block' style={draggableStyle}>
+        <ListItem.Root ref={forwardedRef} id={id} classNames='block' style={draggableStyle}>
           <div
             role='none'
             className={mx(
               surfaceElevation({ elevation: 'group' }),
               inputSurface,
               hoverableControls,
-              'flex rounded',
+              'flex',
+              index === 0 && 'rounded-t',
+              index === count - 1 && 'rounded-b',
```

Comment:
Prefer logical properties — instead of physical `t` use logical `bs`, instead of physical `b` use logical `be`.